stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/View.kt

```diff
@@ -0,0 +1,11 @@
+package io.getstream.chat.android.ui.utils.extensions
+
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.constraintlayout.widget.ConstraintSet
+
+internal inline fun ConstraintLayout.updateConstraints(actions: ConstraintSet.() -> Unit) {
+    val set = ConstraintSet()
+    set.clone(this)
+    set.actions()
+    set.applyTo(this)
+}
```

Comment:
This is already there in `stream-chat-android`, we could make it `@InternalStreamChatApi` to reuse it here